test/google-calendar-api.spec.js

```diff
@@ -13,64 +13,117 @@ var googleApiWrapper = rewire('../src/google-calendar-api');
 var fakeCalendarApi = {};
 
 googleApiWrapper.__set__({
-	credentials: Promise.resolve({}),
+	credentials: sinon.stub().returns(Promise.resolve({})),
 	calendar: fakeCalendarApi
 });
 
 describe('Google Calendar API', () => {
-	var clientEmail, targetCalendar;
+	var clientEmail, targetCalendar, date;
 
 	before(() => {
 		clientEmail = chance.email();
 		targetCalendar = chance.email();
 		process.env.GOOGLE_CLIENT_EMAIL = clientEmail;
 		process.env.GOOGLE_TARGET_CALENDAR = targetCalendar;
+		date = new Date(chance.date({ year: 2017 }));
```

Comment:
Seems to be the case. I think your comment for using moment across the board instead of native Date objects is a good idea. I don't think moment has a random generator so I'll probably construct moment objects in the tester classes by supplying chance.date